elements/sdk/eos-knowledge-lib.bst

```diff
@@ -0,0 +1,38 @@
+kind: autotools
+
+build-depends:
+- freedesktop-sdk.bst:public-stacks/buildsystem-autotools.bst
+- sdk-build-depends/eos-hotdoc-theme.bst
+- sdk-build-depends/hotdoc-modular-framework.bst
+- sdk-build-depends/hotdoc.bst
+- sdk-build-depends/jasmine-gjs.bst
+
+depends:
+- freedesktop-sdk.bst:components/gstreamer.bst
+- gnome-sdk.bst:sdk/WebKitGTK.bst
+- gnome-sdk.bst:core/evince.bst
+- gnome-sdk.bst:sdk/gjs.bst
+- sdk-depends/emeus.bst
+- sdk-depends/eos-knowledge-content-renderer.bst
+- sdk-depends/mathjax.bst
+- sdk-depends/maxwell.bst
+- sdk-depends/python-polib.bst
+- sdk-depends/python-pyyaml.bst
+- sdk-depends/sassc.bst
+- sdk/eos-sdk.bst
+- sdk/libdmodel.bst
+
+variables:
+  conf-local: |
+    --disable-js-doc \
+    --disable-documentation \
+    --with-mathjax-dir="%{datadir}/javascript/mathjax"
+
+environment:
+  HOTDOC: "%{bindir}/python3 %{bindir}/hotdoc"
+
+sources:
+- kind: git_tag
+  url: github_com:endlessm/eos-knowledge-lib.git
+  track: sdk-6
```

Comment:
This git branch seems to be used in lots of places. Perhaps it could be a variable? That would make bumping to the next release much easier.

Reply:
Yeah, good call. We can add that to project.conf - just can't be a build-time option.

Reply:
Now I remember, I tried doing that but the git_tag plugin doesn't resolve variable references there. (I imagine BuildStream is employing a large hammer here to avoid some mishaps, but maybe they'd be cool making the track field do variable substitution...).